### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by jakubMikolajczykTech**
## Click Up Task Link ⏰

[Paste here task description from ClickUp](Paste here link to ClickUp task)

## What has been done 😄

// Describe what have you done shortly //

### CODE REVIEW

**Review by PawelP2001 — APPROVED** [review]

**Review by klaudia-klis — CHANGES_REQUESTED** [review]
Remove unnecessary comments.

File: src/components/Main/ResponsiveMain.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      margin: 56px 0 32px;
+    }
+  }
+
+  .section-1 .section__button.responsive {
+      display: none;
+  }
+
+  .section-1 .section__image.responsive {
+      width: 100% !important;
```
Comment (emil-owczarek) [inline src/components/Main/ResponsiveMain.scss:74]: using !important is not the best practice

File: src/components/utils/fonts/fonts.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('Montserrat-SemiBold.ttf') format('truetype');
+  font-weight: 600;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
```
Comment (emil-owczarek) [inline src/components/utils/fonts/fonts.scss:16]: Did you mean "Monst"? These are different fonts. We can find them here:
https://github.com/mate-academy/product_catalog/tree/main/public/fonts

**Review by emil-owczarek — CHANGES_REQUESTED** [review]
[Nagranie ekranu z 2023-10-31 20-30-24.webm](https://github.com/pl-fe-may23-codecreators/product-catalog/assets/131079210/db6eac5d-ae5c-4604-973a-4539a2c290a0)

It doesn't work the way we want it to. The whole page looks pretty nice, but is not functional. I think it would be better at this point, to implement real data instead of divs.

**Comment by jakubMikolajczykTech** [comment]
> Nagranie.ekranu.z.2023-10-31.20-30-24.webm 
> It doesn't work the way we want it to. The whole page looks pretty nice, but is not functional. I think it would be better at this point, to implement real data instead of divs.

**Pull request closed without merging**